### DISCUSSION TITLE
是否支持doris

### DISCUSSION THREAD

**Discussion by calvin2020** [hhyo/Archery#2175] [Q&A]
Hi:
请教下，目前archery是否支持doris，我把doris添加到archery，查询没有发现啥问题，执行DML以及DDL时会出现一些报错

**Comment by zuoliansheng** [comment]
代码已经适配好了。可惜我们公司不用

**Reply by zuoliansheng** [reply]
要不我贡献出来大家用吧

**Comment by zuoliansheng** [comment]
https://github.com/zuoliansheng/archery-starrocks.git
